tests/src/lookup-set.js

```diff
@@ -0,0 +1,41 @@
+import {
+    NearContract,
+    NearBindgen,
+    call,
+    view,
+    LookupSet
+} from 'near-sdk-js'
+
+@NearBindgen
+class LookupSetTestContract extends NearContract {
+    constructor() {
+        super()
+        this.lookupSet = new LookupSet('a');
+    }
+
+    deserialize() {
+        super.deserialize();
+        this.lookupSet = Object.assign(new LookupSet, this.lookupSet);
+    }
+
+    @view
```

Comment:
Wouldn't  any non-call be a view? Or are any method without a decorator considered private?